export/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ExportAdapter.java

```diff
@@ -120,8 +120,8 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
 
     Subscriber subscriber = null;
     try {
-      SubscriptionName subscriptionName =
-          SubscriptionName.of(Flags.projectId, Flags.subscriptionId);
+      ProjectSubscriptionName subscriptionName =
```

Comment:
Why the change of class here? Java 11 compatibility?

Reply:
More specifically changes in 'com.google.cloud:google-cloud-pubsub:1.89.0' (I couldn't keep old version due to dependencies conflict).
From what I understand it's exactly same thing, but renamed.

Reply:
I see, could you please test manually that the export adapter still works after this change?
We should probably have an integration test for the export adapter if we are making changes to it, but it's not really a high priority task right now.

Reply:
I already tested it manually, that's the point where I had to replace old stowrs implementation with jetty one.